d2bs/kolbot/libs/bots/Andariel.js

```diff
@@ -6,6 +6,9 @@
 
 module.exports =  function(Config, Attack, Pickit, Pather, Town, Misc) {
 	Town();
+	if (me.getStat(sdk.stats.Poisonresist) < 75) {
+		Town.stackPotions('yps');
```

Comment:
We might want to include potion types in the sdk to make this more readable.

`sdk.potiontypes.antidote`

I refrained from adding this myself because I'm unsure what exactly we are adding to the sdk.